src/components/carousel/carousel.html.twig

```diff
@@ -121,6 +121,7 @@
       >
         {% include '@ecl/banner/banner.html.twig' with item|merge({
           size: _size,
+          full_width: _full_width,
```

Comment:
mmh..it seems to work fine, but we are nesting an hack inside another hack, is this the way to solve a "non issue", just an inconsistency, i wonder? (also, are we sure that the situation in the banner is correct, that this override in the padding is really needed?) I would probably prefer only having a css rule as we have in the banner that we would possibly remove in the future, since this has no other scope that applying that rule also when the banner is used in a carousel.